app/controllers/application_controller.rb

```diff
@@ -1,5 +1,15 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  protect_from_forgery with: :exception
+  before_action :authenticate_user!, :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up) do |u|
+      u.permit(:name, :email, :bio, :photo, :password, :password_confirmation)
+    end
+    devise_parameter_sanitizer.permit(:account_update) do |u|
```

Comment:
- [ ] I got a blank page after running `rails s`. It is required that your default route should be set to display sign in page. At this moment, that does not happen. Kindly look into it and fix.

### Your Page

![page](https://github.com/MAleemH/Blog-App/assets/26597432/59c2e589-11d2-4769-b8bb-021511c186c0)